From e3a2eec01694a5a7cfa8c3fb0a561f9f74f3a21c Mon Sep 17 00:00:00 2001
From: Nick Terrell <terrelln@fb.com>
Date: Thu, 30 Jul 2020 12:08:41 -0700
Subject: [PATCH 08/10] Documentation: dontdiff: Add zstd compressed files

For now, that's arch/x86/boot/compressed/vmlinux.bin.zst but probably
more will come, thus let's be consistent with all other compressors.

Signed-off-by: Nick Terrell <terrelln@fb.com>
---
 Documentation/dontdiff | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/dontdiff b/Documentation/dontdiff
index ef9519c32..e361fc95c 100644
--- a/Documentation/dontdiff
+++ b/Documentation/dontdiff
@@ -55,6 +55,7 @@
 *.ver
 *.xml
 *.xz
+*.zst
 *_MODULES
 *_vga16.c
 *~
-- 
2.28.0.rc2.1.g3d20111cbd

